manifests/global_hook.pp

```diff
@@ -8,8 +13,11 @@
 #
 # @param type The custom hook type. Should be one of pre-receive, post-receive, or update.
 # @param custom_hooks_dir The GitLab shell repos path. This defaults to '/opt/gitlab/embedded/service/gitlab-shell/hooks' if not present.
-# @param content Specify the custom hook contents either as a string or using the template function. If this paramter is specified source parameter must not be present.
-# @param source Specify a file source path to populate the custom hook contents. If this paramter is specified content parameter must not be present.
+# @param content Specify the custom hook contents either as a string or using the template function. If this paramter is specified source
+#        parameter must not be present.
+# @param source Specify a file source path to populate the custom hook contents. If this paramter is specified content parameter must not
+#        be present.
```

Comment:
please also undo this line breaks. they just add noise to the diff and make it harder to read and quite tricky to cherry-pick/revert the commit.